### ISSUE TITLE
Giuseppe Scaramuzzino - Tutorial Guy

### ISSUE THREAD

**Issue by dandreadis** [quarkusio/quarkus-insights#44]
https://www.youtube.com/channel/UCV_YnKkwrO1UW6tbASuNaGw

**Comment by maxandersen** [comment]
@insectengine i pinged Giuseppe who is up for coming on - he is thinking about possible topic.

**Comment by GiuseppeScaramuzzino** [comment]
Hello, 
thank you for this opportunity.
I thought about 2 different topics to talk about:
A) Share my story about how and why I have decided to start a YouTube Channel and talk about Quarkus
B) Share how it is easy to create a simple project with Quarkus applying the MicroProfile specifications.

More in detail:
A) I can talk about my profile, my experience as a software engineer (always with Spring), and when, how, and why push me to open a YouTube Channel. Which are my learning resources etc.
B) Quarkus respects and implements the MicroProfile specifications. 
This is an important aspect for a developer who decides to approach this framework for the first time, differentiating Quarkus from other Java frameworks in the market, such as Spring.
First, I would like to give a quick explanation of what the MicroProfile specifications are.
Then, in the core part of the talk, I would like to show how to create a simple project with Quarkus (just a single endpoint) applying the MicroProfile specifications (some of them).  


Let me know what you think. I am open to change or modify these topics.

**Comment by dandreadis** [comment]
Sounds like a plan, to me.

**Comment by GiuseppeScaramuzzino** [comment]
Both of them or 1 of them?

**Comment by dandreadis** [comment]
Yeah both. Let's here it from Max though, I'm not sure who is going to host you.

**Comment by GiuseppeScaramuzzino** [comment]
Sure, let's hear it from Max. I need also to know, how much time for the talk and how to organize it.
Thanks by the way

**Comment by insectengine** [comment]
hello @GiuseppeScaramuzzino,

While you and @maxandersen  are sorting out the topic, I'd like to put you on the schedule. Does either Monday, August 9th or the 16th. Does either of those dates work for you? 

To answer your question, the episode lasts for one hour.

**Comment by maxandersen** [comment]
I think both topics are interesting - and i would ask about A even if we did B as primary.

lets do 30% time on A and 70% on B ?

**Comment by GiuseppeScaramuzzino** [comment]
Sound good for me.
As date I prefer Monday, August 9th.
To recap I can plan the talk in this way:

- 10 min Present my profile and share the story why I have decided to open a YouTube Channel talking about Quarkus
- 10 min Presentation about MicroProfile specification
- 30 min Demo on how to  create a simple project with Quarkus applying the following MicroProfile specifications: CDI / JAX-RS / JSON-P / JSON-B / RestClient (calling an external service) / OpenAPI (quickly document my API) / Fault Tolerance (with just the Timeout and FallbackMethod) / Health and if I have time JWT
- 5/10 min Q&A and what's next...

If you agree and confirm, I will start to prepare the talk.

**Comment by maxandersen** [comment]
Looks good - heads up that we tend to adapt the show based on questions be asked as they come in not wait to end. Means don't plan for super tight On time demo. Might not make it through it all :) Does that work for you ?

**Comment by GiuseppeScaramuzzino** [comment]
Yes, good for me.
I will think about the demo more in detail and I will share with you the plan. So we can be sure to complete it in time.

**Comment by insectengine** [comment]
@GiuseppeScaramuzzino - Perfect... I'll put you on the schedule for August 9th.

@maxandersen  - Suggested title for the episode?

**Comment by insectengine** [comment]
Title : "Quarkus projects using MicroProfile specs with Giuseppe Scarauzzino"

Description: "Join us as Giuseppe Scarauzzino comes by to discuss his youtube channel, tutorials and building Quarkus projects using MicroProfile specifications"

**Comment by GiuseppeScaramuzzino** [comment]
Thanks for the Title and description. They are perfect.
A small change inside the description: Scaramuzzino not Scarauzzino  :)

**Comment by insectengine** [comment]
Sorry about that... fixed!

**Comment by insectengine** [comment]
@GiuseppeScaramuzzino  - Can you share your email with me (jcobb@redhat.com) so I can send you an invite to next week's Quarkus Insights session?

**Comment by GiuseppeScaramuzzino** [comment]
Sure I will do it.
I am finishing the slide that I prepared along side the Demo.
https://docs.google.com/presentation/d/1dNZ_Lw-xX927woEL2v0xNQ9LqBBe-LbmQVD3woUjLTk/edit?usp=sharing
Here the details about the talk:
- I will give a very brief introduction about the history MicroProfile specs
- I will show how simple it is to create a Quarkus application from scratch (just 1 endpoint) that uses most of the MP specs (the green one inside the slide, still don't know if I will have the time for JWT)
- we can discuss about my YouTube channel and what you can find and why I decided to open it
- Q&A

What do you think?

**Comment by GiuseppeScaramuzzino** [comment]
Hello,
what kind of questions usually the people ask?
Is there something in particular that you want to know (about me or else in general)?
In this case I can be prepared.

Thank you for your help.

**Comment by ebullient** [comment]
There isn't a "usually".. it depends on how the conversation goes. We try to keep it interesting so people do ask questions.. so it might be a preference question (why do you do this and not that), sometimes it is a "how/can you do something like this" kind of question..

**Comment by GiuseppeScaramuzzino** [comment]
Hello everyone,
thank you so much for the opportunity, I have enjoyed  the session!!!

**Comment by GiuseppeScaramuzzino** [comment]
Here the source code of the Demo:
https://github.com/GiuseppeScaramuzzino/quarkus-microprofile-specs-demo

https://github.com/GiuseppeScaramuzzino/quarkus-microprofile-jwt-demo
